### DISCUSSION TITLE
Is my canvas rendering okay?

### DISCUSSION THREAD

**Discussion by ghost** [iddan/react-native-canvas#299] [Q&A] [answered]
Hello. I was looking into using this wonderful tool. Then I came across a problem... 

![canvas not rendering correctly](https://user-images.githubusercontent.com/74458502/196281892-715e51e1-718b-43ff-bfef-c710e2d4cfd0.png)

It's hard to explain but I will try...
I set the canvas to be 300x300. And as you can tell, based on the background, that is true (or I guess that is true for the ouster layer). However, the DOM canvas itself if at 300x150, no matter what I do..

I see these instructions...
<img width="488" alt="image" src="https://user-images.githubusercontent.com/74458502/196280770-866c5f37-33ff-433f-937a-85ff7f418ce6.png">

However, I am not understanding how to do this. Can you show me some examples? Also, is this the reason why the canvas size didn't change? Please help me @iddan

**Comment by ghost** [comment, marked as answer]
Oh, never mind...

I just seen the answer from the example... 
https://github.com/iddan/react-native-canvas/blob/master/example/App.js

Thank you very much. I will close this out and maybe if someone else got confused like me they can see this chat someday. Thank you for this awesome product!

**Comment by mobreviews** [comment]
@kennyfully88 I don't follow.  I had a look at that page but from there all I can see is how to open multiple canvas elements not how to increase the size of a single element?

**Comment by mobreviews** [comment]
never mind I worked it out.  I was trying to apply the height and width to the context not the canvas.
